MANIFEST.in

```diff
@@ -1,23 +1,59 @@
+# Base requirements and contstraints
 include requirements/base.in
+include requirements/constraints.in
+
+# Unstructured library extras
+include requirements/extra-csv.in
+include requirements/extra-docx.in
+include requirements/extra-epub.in
+include requirements/extra-markdown.in
+include requirements/extra-msg.in
+include requirements/extra-odt.in
+include requirements/extra-paddleocr.in
+include requirements/extra-pandoc.in
+include requirements/extra-pdf-image.in
+include requirements/extra-pptx.in
+include requirements/extra-xlsx.in
 include requirements/huggingface.in
-include requirements/ingest-s3.in
-include requirements/ingest-azure.in
-include requirements/ingest-discord.in
-include requirements/ingest-github.in
-include requirements/ingest-gitlab.in
-include requirements/ingest-reddit.in
-include requirements/ingest-notion.in
-include requirements/ingest-slack.in
-include requirements/ingest-wikipedia.in
-include requirements/ingest-google-drive.in
-include requirements/ingest-gcs.in
-include requirements/ingest-elasticsearch.in
-include requirements/ingest-opensearch.in
-include requirements/ingest-dropbox.in
-include requirements/ingest-box.in
-include requirements/ingest-onedrive.in
-include requirements/ingest-outlook.in
-include requirements/ingest-confluence.in
-include requirements/ingest-airtable.in
-include requirements/ingest-sharepoint.in
-include requirements/ingest-mongodb.in
+
+# Ingest extras
+include requirements/ingest/airtable.in
+include requirements/ingest/azure-cognitive-search.in
+include requirements/ingest/azure.in
+include requirements/ingest/biomed.in
+include requirements/ingest/box.in
+include requirements/ingest/chroma.in
+include requirements/ingest/confluence.in
+include requirements/ingest/databricks-volumes.in
+include requirements/ingest/delta-table.in
+include requirements/ingest/discord.in
+include requirements/ingest/dropbox.in
+include requirements/ingest/elasticsearch.in
+include requirements/ingest/embed-aws-bedrock.in
+include requirements/ingest/embed-huggingface.in
+include requirements/ingest/embed-huggingface.txt
```

Comment:
Should this .txt be here?

Reply:
Donezo! Dropped a few other `.txt` files too